templates/wizards/projects/asn1acn/file.pro

```diff
@@ -0,0 +1,13 @@
+TEMPLATE = app
+CONFIG += console
+CONFIG -= app_bundle
+CONFIG -= qt
+
+%{JS: if ('%{AddAsnFile}' === 'true' && '%{AddAcnFile}' === 'true')
+        'DISTFILES += \\\\ \n %{AsnFullFileName} \\\\ \n %{AcnFullFileName}'
```

Comment:
I made it this way, because when adding new ASN1 or ACN files to project, it will be added to DISTFILES.